packages/aws-lambda/src/metrics/npmPackageDescription.js

```diff
@@ -16,7 +16,7 @@ exports.currentPayload = undefined;
 exports.activate = function activate() {
   coreUtil.applicationUnderMonitoring.getMainPackageJsonStartingAtDirectory(rootDir.root, (err, pckg) => {
     if (err) {
-      logger.warn('Failed to determine main package json. Reason: ', err.message, err.stack);
+      logger.warn('Failed to determine main package json. Reason: %s, %s', err.message, err.stack);
```

Comment:
issue: Why are we using multiple formats?

> logger.warn('Failed to determine main package json. Reason: ${err.stack});

We should probably use `err.stack || err.message` or ${err.message} ${err.stack} everywhere.